static/src/javascripts/projects/common/modules/identity/sign-in-gate/README.md

```diff
@@ -0,0 +1,16 @@
+# Sign In Gate Test
+## Adding Variants
+
+1. Add the new variant to the sign in gate ab test definition in the `variants` array: `static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate.js`
```

Comment:
Can we add the steps in required to release a new test (ie updating the id, and ing the switch etc)